File: 프로그래머스/unrated/181942. 문자열 섞기/문자열 섞기.java
class Solution {
    public String solution(String str1, String str2) {
        int len = str1.length() + str2.length();
        String answer = "";
        
        for(int i=0;i<str1.length();i++){
            answer += str1.substring(i,i+1);
            answer += str2.substring(i,i+1);
        }  
        return answer; 
    }
}

File: 프로그래머스/0/181942. 문자열 섞기/문자열 섞기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* str1, const char* str2) {
    char* answer = (char*)malloc(strlen(str1) + strlen(str2) * sizeof(char));
    int len = 0;
    
    for(int i=0;i<=strlen(str1);i++){
        answer[len++] = str1[i];
        answer[len++] = str2[i];
    }
    
    return answer;
}